### PR TITLE
Board view integration

### LINKED ISSUE #17: IN FE: Role's board view
- Spy masters get to see master view
- Everyone else gets to see hidden view 
- Make sure its cheat proof

### PR DESCRIPTION
**Pull request by mpmpx**
# Fixes 
- Integrated board view into the layout skeleton

### CODE REVIEW

**Review by sina-jamshidi — APPROVED** [review]
Looks good!

File: client/src/components/Game/Nav.js [hunk 1]
```diff
@@ -113,8 +113,8 @@ const Navbar = props => {
           </Hidden>
           <Hidden smDown>
           <Grid item container className={classes.navEnd}>
-            <Button className={classes.game} variant="outlined">
-              New Game
+            <Button onClick={() => onRestart()} className={classes.game} variant="outlined">
```
Comment (Travmatth) [inline client/src/components/Game/Nav.js:116]: no need to pass a func to call another func,
`onClick={() => onRestart()}` -> `onClick={onRestart}`

File: client/src/components/Game/Nav.js [hunk 2]
```diff
@@ -145,8 +145,8 @@ const Navbar = props => {
           anchorEl={miniMenu}
           open={Boolean(miniMenu)}
           onClose={toggleMenu("mini")}>
-          <MenuItem selected={false} onClick={toggleMenu("mini")}>New
-            Game
+          <MenuItem selected={false} onClick={() => onRestart()}>
```
Comment (Travmatth) [inline client/src/components/Game/Nav.js:148]: ditto from above

File: client/src/pages/Match.js [hunk 3]
```diff
@@ -228,7 +243,7 @@ const Match = withStyles(styles)(({ classes, state, setState, gameID, setGameID,
                 direction="column"
                 className={classes.newGame}
                 justify="center">
-                <Btn on={call("public")} css={classes.public} text="Public"/>
+                <Btn on={() => setErr("not implemented")} css={classes.public} text="Public"/>
```
Comment (Travmatth) [inline client/src/pages/Match.js:246]: ditto from above

**Review by Travmatth — APPROVED** [review]

**Pull request merged**